modules/logs/classes/class-log-events-list-table.php

```
@@ -332,6 +368,39 @@ public function get_action_title( $act, $type, $coloring = false ) {
         return $title;
     }
 
+    /**
+     * Returns the label for a connector term.
+     *
+     * @param string $record  Log data.
+     * @param string $type  Connector term.
+     * @param bool   $coloring Coloring term.
+     * @return string
+     */
+    public function get_event_title( $record, $type, $coloring = false ) {
```

PHPMD (2.15.0)

The method get_event_title has a boolean flag argument $coloring, which is a certain sign of a Single Responsibility Principle violation. (Clean Code Rules)

(BooleanArgumentFlag)